### PR TITLE
fix: Fixed TensorFlow attributes `sum` and `mean`

### LINKED ISSUE #27977: TensorFlow has no attributes `sum` and `mean`
In the following function, the attributes `sum` and `mean` are used
https://github.com/unifyai/ivy/blob/f1fc4cb543615bdc1364821b122c1c2127d4b461/ivy/functional/backends/tensorflow/experimental/losses.py#L25-L30
They should be `reduce_sum` and `reduce_mean`

### PR DESCRIPTION
**Pull request by Sai-Suraj-27**
# PR Description
In the following function, the attributes `sum` and `mean` are used
https://github.com/unifyai/ivy/blob/f1fc4cb543615bdc1364821b122c1c2127d4b461/ivy/functional/backends/tensorflow/experimental/losses.py#L25-L30
They should be `reduce_sum` and `reduce_mean`

## Related Issue
Closes #27977

## Checklist

- [ ] Did you add a function?
- [ ] Did you add the tests?
- [ ] Did you run your tests and are your tests passing?
- [x] Did pre-commit not fail on any check?
- [ ] Did you follow the steps we provided?

### CODE REVIEW

**Review by KareemMAX — COMMENTED** [review]
Can you elaborate more on why `reduce_sum`/`reduce_mean` should be used rather than `sum`/`mean`?

**Comment by Sai-Suraj-27** [comment]
> Can you elaborate more on why `reduce_sum`/`reduce_mean` should be used rather than `sum`/`mean`?

TensorFlow does not have `sum`, `mean` methods. Like `torch.sum()` instead of `sum` and `mean`, 
`reduce_mean` and `reduce_sum` are present in [tensorflow](https://www.tensorflow.org/api_docs/python/tf/math/reduce_sum).

Even in all other places in our codebase, `reduce_mean` and `reduce_sum` are used except in this place.

**Comment by madhav48** [comment]
Is the issue still open? If yes, what was the problem with the last PR regarding this issue, can you please elaborate..

**Comment by Sai-Suraj-27** [comment]
> Is the issue still open? If yes, what was the problem with the last PR regarding this issue, can you please elaborate..

Hey, @madhav48. Yes, this PR/issue is still open, after some internal discussion, It is decided that I will work on fixing the `huber_loss` test. So, I will be working on this PR and update it this week.

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by ivy-seed** [comment]
This PR has been closed because it has been marked as stale for more than 7 days with no activity.
